src/components/BaseModal/BaseModal.jsx

```diff
@@ -0,0 +1,41 @@
+import { ModalBackground, ModalContainer, ModalContainerCloseButWrapper, CloseButImg } from "./BaseModal.styled";
+import closeCross from "./assets/closeCross.svg"
+import { useEffect, useState } from "react";
+
+export const BaseModal = ({ifShown = true}) => {
```

Comment:
We should think how it will be used. There are two possible solutions:
<BaseModal isShown />
or
isShown && <BaseModal />

You chose the first one and it is pretty good, but has some bottlenecks. For example, if you close modal at the first time, you wont be able to show it again with you current solution.

PS. Both solutions will require `closeHandler` from the parent to change `isShown` variable